docs/.vuepress/atomicDEX-sidebar.js

```diff
@@ -115,130 +115,102 @@ let atomicDEXsidebar = {
-          "update_maker_order",
-        ],
-        [
-          "/basic-docs/atomicdex-api-legacy/validateaddress.md",
-          "validateaddress",
-        ],
-        ["/basic-docs/atomicdex-api-legacy/unban_pubkeys.md", "unban_pubkeys"],
-        ["/basic-docs/atomicdex-api-legacy/version.md", "version"],
-        ["/basic-docs/atomicdex-api-legacy/withdraw.md", "withdraw"],
+        ["/basic-docs/atomicdex-api-legacy/batch_requests.md", "batch_requests"],
+        ["/basic-docs/atomicdex-api-legacy/rational_number_note.md", "Note about rational number type"],
+        {
+          title: "Coin Activation",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-legacy/coin_activation.md", "coin activation"],
+            [
+              "/basic-docs/atomicdex-api-legacy/coins_needed_for_kick_start.md",
+              "coins_needed_for_kick_start",
+            ],
+            ["/basic-docs/atomicdex-api-legacy/disable_coin.md", "disable_coin"],
+            ["/basic-docs/atomicdex-api-legacy/get_enabled_coins.md", "get_enabled_coins"],
+            [
+              "/basic-docs/atomicdex-api-legacy/set_required_confirmations.md",
+              "set_required_confirmations",
+            ],
+            [
+              "/basic-docs/atomicdex-api-legacy/set_requires_notarization.md",
+              "set_requires_notarization",
+            ],
+          ]
+        },
+        {
+          title: "Network",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-legacy/get_gossip_mesh.md", "get_gossip_mesh"],
+            ["/basic-docs/atomicdex-api-legacy/get_gossip_peer_topics.md", "get_gossip_peer_topics"],
+            ["/basic-docs/atomicdex-api-legacy/get_gossip_topic_peers.md", "get_gossip_topic_peers"],
+            ["/basic-docs/atomicdex-api-legacy/get_my_peer_id.md","get_my_peer_id",],
+            ["/basic-docs/atomicdex-api-legacy/get_peers_info.md", "get_peers_info"],
+            ["/basic-docs/atomicdex-api-legacy/get_relay_mesh.md", "get_relay_mesh"],
+          ]
+        },
+        {
+          title: "Orders",
+          collapsable: true,
+          children: [
+          ["/basic-docs/atomicdex-api-legacy/best_orders.md", "best_orders"],
+          ["/basic-docs/atomicdex-api-legacy/buy.md", "buy"],
+          ["/basic-docs/atomicdex-api-legacy/cancel_all_orders.md", "cancel_all_orders"],
+          ["/basic-docs/atomicdex-api-legacy/cancel_order.md", "cancel_order"],
+          ["/basic-docs/atomicdex-api-legacy/orderbook.md", "orderbook"],
+          ["/basic-docs/atomicdex-api-legacy/orderbook_depth.md", "orderbook_depth"],
+          ["/basic-docs/atomicdex-api-legacy/orders_history_by_filter.md", "orders_history_by_filter"],
+          ["/basic-docs/atomicdex-api-legacy/order_status.md", "order_status"],
+          ["/basic-docs/atomicdex-api-legacy/my_orders.md", "my_orders"],
```

Comment:
`my_orders` should be right after `cancel_order` :)

Reply:
:facepalm: I need to go back to kindergarten :sweat_smile: